repositories/migrations/20241029013218_scenario_testrun.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE scenario_testrun(
```

Comment:
I'm going to sound like a super annoying guy, but can we please make it `scenario_test_run`? (here and in the other places) 🙏

Reply:
ok no problem.